File: convert.js
import * as tf from '@tensorflow/tfjs'

const model = await tf.loadLayersModel('D:\coding\python\captcha\model.json')

const example = tf.fromPixels('D:\coding\python\captcha\testcases\bin\symbol-Upper-y\id-3mvYL613.png')

const prediction = model.predict(example);

print(prediction)

File: convert.py
import tensorflowjs as tfjs
from keras.models import load_model

MODEL_FILENAME = "D:\coding\python\captcha\captcha_model.hdf5"

model = load_model(MODEL_FILENAME)

tfjs.converters.save_keras_model(model, 'D:\coding\python\captcha')